core/kazoo_stdlib/src/kz_binary.erl

```diff
@@ -213,3 +213,20 @@ ucfirst(<<F:8, Bin/binary>>) -> <<(kz_term:to_upper_char(F)):8, Bin/binary>>.
 
 -spec lcfirst(ne_binary()) -> ne_binary().
 lcfirst(<<F:8, Bin/binary>>) -> <<(kz_term:to_lower_char(F)):8, Bin/binary>>.
+
+-spec pos(char(), binary()) -> non_neg_integer() | -1.
+pos(Char, Bin) ->
+    pos(Char, Bin, 0).
+pos(_, <<>>, _) -> -1;
+pos(Char, <<Char:8, _/binary>>, N) -> N;
+pos(Char, <<_:8, Bin/binary>>, N) ->
+    pos(Char, Bin, N + 1).
+
+-spec closests([char(),...], binary()) -> [{char(), non_neg_integer()}].
```

Comment:
> What is its purpose? Seems like clutter at this point.

Right, because tests are so hard to read.
This function was introduced for the CSV parser but is no longer used.
I did not remove it because I think it can be useful in the future.